File: 50 Coding Challenge Part I/JavaScript/Challenge4.js
// Challenge 4: Print the multiplication tables with numbers from 1 to 10

function multiply(num){
    for (let i = 1; i <= 10; i++){
        let table  =  num + " * " + i + " = " + (i * num);
        console.log(table);
    }
}

for (let i = 1; i <= 10; i++){
    console.log("Multiplication table for " + i);
    multiply(i);
}

File: 50 Coding Challenge Part I/dart/lib/challenge4.dart
// Coding Challenge 4: Print the multiplication tables with numbers from 1 to 10
// @author: mofirojean

void printMultiplicationTable() {
  for (var i = 1; i <= 10; i++) {
    print('\n$i multiplication table');
    printTable(i);
  }
}

void printTable(int num) {
  for (var i = 1; i <= 10; i++) {
    print('$i * $num = ${i * num}');
  }
}
